src/main/java/kitchenpos/menu/domain/Menu.java

```diff
@@ -63,7 +63,7 @@ private void addMenuProduct(final MenuProduct menuProduct) {
     }
 
     private void validatePrice() {
-        if (price.getValue().compareTo(menuProducts.totalPrice()) > 0) {
+        if (price.getValue().compareTo(menuProducts.totalMenuProductPrice()) > 0) {
```

Comment:
getter 를 사용하기 보다는 객체에 메세지를 보내보면 좋을 것 같아요! 😄 
```suggestion
        if (price.compareTo(menuProducts.totalMenuProductPrice()) > 0) {
```

Reply:
계속 이런 부분을 코멘트 다시게 해서 죄송하네요
다음 단계할때는 getter 대신 객체에 메시지를 보낼 수 있는 부분을 찾아보고 전체적으로 적용하도록 하겠습니다 :pray: